### ISSUE TITLE
Bug: lint fail

### ISSUE THREAD

**Issue by alex-lit** [es-joy/jsdoccomment#1]
node v14.17.1

0.9.0-alpha.1 work fine

0.9.0-alpha.2 throw error:
ESLint: Failed to load config "ash-nazg/sauron-node-overrides" to extend from. Referenced from: <project_name>/node_modules/@es-joy/jsdoccomment/.eslintrc.cjs. Please see the 'ESLint' output channel for details.

eslint output:
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: <project_name>/node_modules/@es-joy/jsdoccomment/node_modules/comment-parser/lib/parser/tokenizers/description.js
require() of ES modules is not supported.

**Comment by brettz9** [comment]
Thank you very much for the report.

I've reverted the changes for now in an alpha.3 release. (FWIW, once https://github.com/syavorsky/comment-parser/pull/133 may be merged and released, I should be able to reapply the alpha.2 changes.)

Note that if you are getting this because of `eslint-plugin-jsdoc`, I pushed a fix for that earlier (so that it relied on the alpha.1 version)

Closing but feel free to comment further as needed.
